src/main/java/com/example/zadanie22/MailService.java

```diff
@@ -0,0 +1,42 @@
+package com.example.zadanie22;
+
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MailService {
+
+    private JavaMailSender mailSender;
+    private String companyMail = "test321@poczta.onet.pl";
+
+    public MailService(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+
+    public JavaMailSender getMailSender() {
+        return mailSender;
+    }
+
+    public void setMailSender(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+
+    public String getCompanyMail() {
+        return companyMail;
+    }
+
+    public void setCompanyMail(String companyMail) {
+        this.companyMail = companyMail;
+    }
+
+    public void sendMail(Mail mail, String receiver) {
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setFrom(companyMail);
+        message.setTo(receiver);
```

Comment:
wiadomość powinna trafić do administratora, więc "to" powinno być ustawione na companyMail, a replyTo na receiver.
Wiadomość wysyłamy sami do siebie, a administrator będzie mógł odpowiedzieć tej osobie, która wypełniła formularz w przyszłości.
Ustawienie "to" na receivera byłoby ok w dalszej części zadania, gdzie jest wspomniane o wysyłaniu potwierdzenia do osoby wypełniającej formularz.